fastlane/Fastfile

```diff
@@ -0,0 +1,59 @@
+  # This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
+
+default_platform(:android)
+
+platform :android do
+  desc "Runs all the tests"
+  lane :test do
+    gradle(task: "test")
+  end
+
+  lane :debug do
+    gradle(task: "clean assembleDebug")
+    appcenter
+  end
+
+  lane :release do
+    appcenter_version = appcenter_fetch_version_number(
+      owner_name: "Android-Demo",
+      app_name: "SDK-Demo"
+    )
+    build_number = appcenter_version["build_number"]
+    update_version_name(name: "Release #{appcenter_version["id"].to_i + 1}")
+    gradle(task: "clean assembleRelease")
+    appcenter
+  end
+
+  lane :local_increment do
+    version_code = get_version_code
+    increment_version_code(version_code: version_code)
+  end
+
+  lane :appcenter do
+    apk_path = lane_context[SharedValues::GRADLE_APK_OUTPUT_PATH]
+    appcenter_upload(
+      api_token: ENV["APPCENTER_API_TOKEN"],
+      owner_name: "Android-Demo",
+      app_name: "SDK-Demo",
+      file: apk_path,
+      destinations: ENV["DESTINATION_GROUP"],
+      destination_type: "group",
+      release_notes: (File.read("./CHANGELOG.md") rescue "SDK Fastlane test. Please ignore."),
```

Comment:
If you have a change_log.md file it will use that for the release notes. But yeah we have to populate that either way. Don’t really know how we can get around that because we usually are putting in which tickets the build is for. Not sure how to automatically know that.

Reply:
@ptang-vn We could output `git log` either to the changelog file or we can just pass in the raw output as a string. Presumably we'd want to limit the log to new commits. But that also poses issues in this particular case, since most SDK builds are not committing direct changes to this repository.